data/db-schema/src/main/resources/net/corda/db/schema/config/migration/cpx-creation-v1.0.xml

```diff
@@ -176,6 +176,40 @@
                                  baseTableSchemaName="${schema.name}"
                                  referencedTableSchemaName="${schema.name}"/>
 
+        <createTable tableName="cpk_db_change_log_audit" schemaName="${schema.name}">
```

Comment:
I think it be worth adding a comment **why** we are introducing another table which looks very similar to `cpk_db_change_log`.